src/Browscap/Helper/IteratorHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types = 1);
+namespace Browscap\Helper;
+
+use Psr\Log\LoggerInterface;
+
+class IteratorHelper
+{
+    /**
+     * @param LoggerInterface $logger
+     * @param string          $testKey
+     *
+     * @return array
+     */
+    public function getFiles(LoggerInterface $logger, string $testKey = 'full') : array
```

Comment:
I think `getFiles` is a bit generic for what this method does.  It specifically gets the test files, so perhaps should be named that way.